src/Analytics.php

```diff
@@ -116,6 +116,26 @@ public function fetchTopReferrers(Period $period, int $maxResults = 20): Collect
         });
     }
 
+    public function fetchUserTypes(Period $period): Collection
+    {
+        $response = $this->performQuery(
+            $period,
+            'ga:sessions',
+            [
+                'dimensions' => 'ga:userType',
+            ]
+        );
+
+        $userSessions = collect($response->rows ?? [])->map(function (array $usRow) {
```

Comment:
Let's return the collection directly (like in the other methods).

Use `userRow` instead of `usRow`

Reply:
Submitted the suggested changes.